pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/Auth0LogEventDao.java

```diff
@@ -1,16 +1,31 @@
 package org.broadinstitute.ddp.db.dao;
 
+import static org.slf4j.LoggerFactory.getLogger;
+
+
 import org.broadinstitute.ddp.json.auth0.Auth0LogEvent;
 import org.jdbi.v3.sqlobject.CreateSqlObject;
 import org.jdbi.v3.sqlobject.SqlObject;
+import org.slf4j.Logger;
 
 public interface Auth0LogEventDao extends SqlObject {
 
+    Logger LOG = getLogger(Auth0LogEventDao.class);
+
     @CreateSqlObject
     JdbiAuth0LogEvent getJdbiAuth0LogEvent();
 
-    default long insertAuth0LogEvent(Auth0LogEvent logEvent) {
-        long id = getJdbiAuth0LogEvent().insertAuth0LogEvent(
+    /**
+     * Check if auth0 log event exist: if not then insert it, then find code details info and
```

Comment:
Can you cleanup this file? Don't think this comment is accurate anymore, and don't think we're using `LOG`.

Reply:
Cleaned up.